src/group_theory/order_of_element.lean

```diff
@@ -390,21 +594,34 @@ begin
 end
 
 lemma order_eq_card_powers [decidable_eq α] {a : α} :
-order_of a = fintype.card (submonoid.powers a : set α) :=
+  order_of a = fintype.card (submonoid.powers a : set α) :=
 begin
   refine (finset.card_eq_of_bijective _ _ _ _).symm,
-  { exact λn hn, ⟨pow a n, ⟨n, rfl⟩⟩ },
+  { exact λn hn, ⟨a ^ n, ⟨n, rfl⟩⟩ },
   { rintros ⟨_, i, rfl⟩ _,
     exact ⟨i % order_of a, mod_lt i (order_of_pos a), subtype.eq pow_eq_mod_order_of.symm⟩ },
   { intros, exact finset.mem_univ _ },
   { intros i j hi hj eq,
     exact pow_injective_of_lt_order_of a hi hj ( by simpa using eq ) }
 end
 
+lemma add_order_of_eq_card_multiples [decidable_eq H] {x : H} :
+  add_order_of x = fintype.card (add_submonoid.multiples x : set H) :=
+begin
+  rw ← order_of_of_add_eq_add_order_of,
+  rw order_eq_card_powers,
```

Comment:
Can you combine these `rw`s? Or if the first one closes the main goal and adds a subgoal, could you put `{}`-braces around the second part of the proof?
```suggestion
  rw [← order_of_of_add_eq_add_order_of, order_eq_card_powers],
```